### DISCUSSION TITLE
L2 shows 0A when L1 and L3 has a high amp draw.

### DISCUSSION THREAD

**Discussion by putte64** [UtilitechAS/amsreader-firmware#670] [General]
While charging my car with single phase 32A, the graph in AMS reader will show 0A in L2 and high amp draw in L1 and L3. Most likely my TWC v2 draws from L1 and L3 so high amperage is expected, but there is still heaters in the house connected to L2 and it should show some amps there to. Disconnecting the car would make L2 show up with a value almost same as L1 and L3.

Is this a bug in Aidon, AMS reader hardware or AMS reader software or am I simply missing some setting somewhere?

![IMG_0424](https://github.com/UtilitechAS/amsreader-firmware/assets/2694088/6b85ccfa-b4ae-4320-b472-a8ae2a72a56c)

**Comment by ArnieO** [comment]
For some reason, some meter models indeed report 0A on L2, see this document:
https://github.com/UtilitechAS/amsreader-firmware/blob/master/doc/Norway/OBIS%20List%20Information%20-%20AIDON.pdf
![image](https://github.com/UtilitechAS/amsreader-firmware/assets/10295178/acd533e0-f642-4f78-b3c4-d5b513ab755b)

Could you check on your info-page and see if your Aidon model is the one indicated in the above table?

In those cases, I believe the firmware should estimate the current on L2 (calculated from L1 and L2). But it is apparently not done in your case. **Can you see a reason for that, @gskjold ?**

**Reply by gskjold** [reply]
I think the formula used for calculation is just way off, and it looks like the higher current draw, the worse it gets. We should have a look at that formula one day

**Reply by ArnieO** [reply]
OK, I'll look into the equation. 
But in this case, I think zero on L2 is a logical error, not a formula problem?

**Reply by ArnieO** [reply]
@putte64 : 
- Is it possible that (close to) zero was indeed correct here? 
- Please provide screendump of the "Meter" section on your Info-page.

**Reply by gskjold** [reply]
According to the calculation used, the data from above screenshot would give negative 19 amps on calculated L2, so definitely formula error

**Comment by putte64** [comment]
Screenshot of Meter-page and picture of meter version.

L2 is a lot lower, but not 0. If amp-draw on L1-L3 was lowered, L2 would show some amps again.


![IMG_0425](https://github.com/UtilitechAS/amsreader-firmware/assets/2694088/5633b326-589b-4bb5-b929-e82ac8fe325b)
![20231106_155942_Original](https://github.com/UtilitechAS/amsreader-firmware/assets/2694088/e3a9356d-ff22-454f-9328-1e56b044f881)
